apps/ff-site/src/app/orbit/services/airtable.ts

```diff
@@ -7,21 +7,20 @@ import {
   AIRTABLE_ORBIT_EVENTS_CONFIG,
   APPROVED_STATUS_VALUE,
 } from '../constants/airtableOrbitEventsConfig'
+import { AirtableRecordSchema } from '../schema/AirtableRecordSchema'
 
-const airtable = new Airtable({ apiKey: process.env.AIRTABLE_READ_ONLY_TOKEN })
+const airtable = new Airtable({
+  apiKey: process.env.AIRTABLE_READ_ONLY_TOKEN,
+  requestTimeout: 3_000,
```

Comment:
Nit but best practices dictate that timeouts, limits, URLs, API keys should all live in config/env and not source code.

`requestTimeout: Number(process.env.AIRTABLE_REQUEST_TIMEOUT ?? 3000)`

Which would allow ops/devs to ovverride via `AIRTABLE_REQUEST_TIMEOUT=5000` (or shorter for CI tests)